src/components/dialogs/StandardDialog.svelte

```diff
@@ -5,68 +5,99 @@
  -->
 
 <script lang="ts">
-  import { createDialog } from 'svelte-headlessui';
+  import { CreateDialogProps, createDialog, createSync, melt } from '@melt-ui/svelte';
+  import { fade, scale } from 'svelte/transition';
+  import { quintOut } from 'svelte/easing';
   import CloseIcon from 'virtual:icons/ri/close-line';
-  import Transition from 'svelte-transition';
   import { t } from '../../i18n';
   import IconButton from '../IconButton.svelte';
   export let hasCloseButton = true;
+  export let createDialogPropsOverride: CreateDialogProps = {};
```

Comment:
Can we just have `closeOnOutsideClick` and `closeOnEscape` with defaults. 

Then in the createDialog call
{ 
 `closeOnOutsideClick` ,
`closeOnEscape`,
...
}

I think this is cleaner for passing the props in.

Reply:
Have made the change here: https://github.com/microbit-foundation/ml-trainer/pull/171/commits/8c7e37c43dbc341168dbca9ba785c6e1dd19ad52